pkg/config/config.go

```diff
@@ -59,18 +59,31 @@ func getConfigurationFilePathFromEnvironment() string {
 // as the reference to the interactive.Oc instance, the reference to the test configuration, and the default network
 // IP address.
 type Container struct {
-	ContainerConfiguration  configsections.ContainerConfig
-	Oc                      *interactive.Oc
+	configsections.ContainerConfig
```

Comment:
do you need a variable here ?

Reply:
i did it on purpose so test code is more readable, e.g. cut.PodName instead of cut.ContainerConfig.ContainerIdentifier.PodName

Reply:
Also take a look at the latest commit. I redefined HasDebugPod() so I don't have to do nil check everywhere just to avoid panic on minikube